### PR TITLE
fix(design tokens): missing CSS var for shadow hover error

### PR DESCRIPTION
**Pull request by pflynn-virtru**
changes: _rev_

### Proposed Changes
`.container p` rule removed. no var available, and rule above already handles it
`--vds-shadow-hoverError` corrected

### Checklist

- [ ] I have added or updated unit tests
- [ ] I have added or updated integration tests (if appropriate)
- [ ] I have added or updated documentation / readme (if appropriate)
- [x] I have verified that my changes have not introduced new lint errors
- [ ] I have updated the change log

### CODE REVIEW

File: lib/styles/build/css/design_tokens.css [hunk 1]
```diff
@@ -97,7 +97,7 @@
   --vds-shadow-focus: var(--vds-shadow-hover);
   --vds-shadow-pressed: 0 1px 4px 1px rgba(45, 50, 59, 0.3);
   --vds-shadow-hover-error: 0 1px 4px 1px rgba(81, 88, 100, 0.3);
-  --vds-shadow-focus-error: var(--vds-shadow-hoverError);
+  --vds-shadow-focus-error: var(--vds-shadow-hover-error);
```
Comment (avkvirtru) [inline lib/styles/build/css/design_tokens.css:100]: This entire file and its JS counterpart are actually generated from a build tool called [style dictionary](https://virtru.github.io/virtuoso-design-system/?path=/docs/basics-design-tokens--page). If we need to make changes, they should be in the token JSON file: 
https://github.com/virtru/virtuoso-design-system/blob/master/lib/styles/style_dict/tokens/shadow.json

I'm not clear on why the variable name changes here…

Comment (avkvirtru) [reply lib/styles/build/css/design_tokens.css:100]: May be faster to simply duplicate the focus error token with the hover error value:

```json
"focusError":  { "value": "0 1px 4px 1px rgba(81, 88, 100, 0.3)" }
```

Comment (avkvirtru) [reply lib/styles/build/css/design_tokens.css:100]: @pflynn-virtru see if you can update shadow.json and (hopefully) follow these instructions to generate the token files (CSS vars and JS exports): https://github.com/virtru/virtuoso-design-system/blob/master/README.md#design-tokens

**Comment by avkvirtru** [comment]
@jrschumacher could you review the container fix here?

File: lib/styles/build/css/design_tokens.css (hunk 1 above)
Comment (avkvirtru) [reply lib/styles/build/css/design_tokens.css:100]: @pflynn-virtru lmk if you need any help doing the above. Excited to fix this bug you found (and test out our documentation for design tokens).

**Comment by avkvirtru** [comment]
@pflynn-virtru @jrschumacher reminder to sync up on this when you two can—would love to get this fixed & merged in

File: lib/styles/build/css/design_tokens.css (hunk 1 above)
Comment (avkvirtru) [reply lib/styles/build/css/design_tokens.css:104]: @pflynn-virtru @jrschumacher sorry for the delay on this. I fixed the problem at the source [in this commit](https://github.com/virtru/virtuoso-design-system/pull/72/commits/3178bf87f155df56dea352e81a050a5c09312ff3).

**Review by avkvirtru — APPROVED** [review]
LG2M 👍

**Comment by avkvirtru** [comment]
@jrschumacher can you sanity check this when you have a moment?

**Review by jrschumacher — APPROVED** [review]
@pflynn-virtru looks good to me

**Comment by virtru-github-public** [comment]
:rocket: PR was released in [`v1.14.1`](https://github.com/virtru/virtuoso-design-system/releases/tag/v1.14.1) :rocket:

**Pull request merged**